src/org/parosproxy/paros/extension/report/ReportGenerator.java

```diff
@@ -126,7 +129,8 @@ public static File stringToHtml(String inxml, String infilexsl, String outfilena
 					bw = new BufferedWriter(new FileWriter(outfilename + "-orig.xml"));
 					bw.write(inxml);
 				} catch (IOException e2) {
-					logger.error(e.getMessage(), e);
+					logger.error(Constant.messages.getString("report.write.diskspace.error.log.message"), e);
+					JOptionPane.showMessageDialog(View.getSingleton().getMainFrame(), Constant.messages.getString("report.write.diskspace.error.log.message"));

```

Comment:
- There might not be a view (i.e. when generating the report through the command line), View.isInitialised() should be checked first;
 - Probably better to leave the parent null, it might be other dialogue;
 - The other IOException catch statements (the ones when writing) should have this warning too.